Cargo.toml

```diff
@@ -123,7 +123,7 @@ wgpu-example = { version = "0.18.0", path = "./examples/common" }
 wgpu-macros = { version = "0.18.0", path = "./wgpu-macros" }
 wgpu-test = { version = "0.18.0", path = "./tests" }
 wgpu-types = { version = "0.18.0", path = "./wgpu-types" }
-winit = { version = "0.29.2", features = [ "android-native-activity" ] }
+winit = { version = "0.29.3", features = [ "android-native-activity" ] }
```

Comment:
We can leave this at 0.29 - and just update the lock file.

Reply:
so, you mean, I can just remove the third number (here the `3`)?

Reply:
Yes - it's equivalent to writing 0.29.0

Reply:
Done!